test/e2e/tests/test_hpo.py

```diff
@@ -86,5 +82,5 @@ def test_create_hpo(self, xgboost_hpojob):
         _, deleted = k8s.delete_custom_resource(reference)
         assert deleted is True
 
-        hpo_sm_desc = get_sagemaker_hpo_job(hpo_job_name)
+        hpo_sm_desc = get_sagemaker_hpo_job(sagemaker_client, hpo_job_name)
```

Comment:
we cannot keep using the same boto3 client object `sagemaker_client`

the session in the object might expire 

when the credentials are refreshed the new credentials won't be picked up if we don't create new session (or new client like we used to before you made this change)

Reply:
as discussed offline, good suggestion but needs to be in a separate PR
hpo was an doing it differently
this PR is just moving the fixture to a different file not changing the existing behaviour